website/docs/r/federated_settings_identity_provider.html.markdown

```diff
@@ -49,7 +49,6 @@ In addition to all arguments above, the following attributes are exported:
 ### FederatedSettingsIdentityProvider
 
 * `okta_idp_id` - Unique 20-hexadecimal digit string that identifies the IdP.
-* `id` - Unique 24-hexadecimal digit string that identifies the IdP.
```

Comment:
can you help me understanding? Why are we removing the `id` field? All this time I thought the problem was `okta_` id..

Reply:
Fixed, removing the `id`, but now returning `idp_id`